boards/arm/cxd56xx/common/src/cxd56_crashdump.c

```diff
@@ -98,15 +98,14 @@ static void copy_reverse(stack_word_t *dest, stack_word_t *src, int size)
  * Name: board_crashdump
  ****************************************************************************/
 
-void board_crashdump(uintptr_t currentsp, void *tcb,
-                     const char *filename, int lineno)
+void board_crashdump(uintptr_t sp, struct tcb_s *tcb,
+                     const char *filename, int lineno,
+                     const char *msg)
```

Comment:
Do we need to add something like `UNUSED(msg);`?

Reply:
Since the compiler didn't generate the warning, I prefer to remove the unnecessary but ugly UNUSED(xxx).